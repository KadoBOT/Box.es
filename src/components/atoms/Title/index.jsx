import React from 'react'
import styled from 'styled-components';

const Title = styled.div`
  font-size: ${props => props.subheader ? '14px' : '32px'};
  font-weight: ${props => props.subheader ? 100 : 'bold'};
  color: ${props => props.subheader ? '#b3b3b3' : 'rgb(59, 59, 59)'};
  display: -webkit-box;
  overflow: hidden;
  -webkit-box-orient: vertical;
  -webkit-line-clamp: 2;
  line-height: ${props => props.subheader ? '20px' : '40px' };
  max-height: 80px;
`

export default Title
