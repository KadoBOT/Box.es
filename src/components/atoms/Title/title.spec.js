import React from 'react'
import renderer from 'react-test-renderer';
import Title from './index'

it('should render the Title', () => {
    const wrapper = renderer.create(
      <Title />
    );
    expect(wrapper.toJSON()).toMatchSnapshot();
});

it('should render the Subheader Title', () => {
    const wrapper = renderer.create(
      <Title subheader />
    );
    expect(wrapper.toJSON()).toMatchSnapshot();
});
