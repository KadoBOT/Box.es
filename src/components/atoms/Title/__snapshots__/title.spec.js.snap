exports[`test should render the Subheader Title 1`] = `
<div
  className="logIZ" />
`;

exports[`test should render the Title 1`] = `
<div
  className="hKXDia" />
`;
